src/docfx/models/GitUserInfo.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Docs.Build
+{
+    internal class GitUserInfo
+    {
+        [JsonProperty("profile_url")]
```

Comment:
For v3, the output model does not need [JsonProperty], it will automatically be camel cased

Reply:
nit: use `GitHubUser` and follow the terms used by `GitHub` in their REST API. I remember they call it `login`, `name`, `avatar`

Reply:
I think you mean [https://developer.github.com/v3/users/](https://developer.github.com/v3/users/) . Seems the naming is hard to understand under docfx's scenario.

---
In reply to: [197005200](https://github.com/dotnet/docfx/pull/2912#discussion_r197005200) [](ancestors = 197005200)